README.md

```diff
@@ -2432,6 +2432,39 @@ config.properties['hazelcast.client.statistics.period.seconds'] = 4;
 
 After enabling the client statistics, you can monitor your clients using Hazelcast Management Center. Please refer to the [Monitoring Clients section](https://docs.hazelcast.org/docs/management-center/latest/manual/html/index.html#monitoring-clients) in the Hazelcast Management Center Reference Manual for more information on the client statistics.
 
+### 7.9.2. Logging Configuration
+
+ To configure a logger, you need to use the `ClientConfig.properties['hazelcast.logging']` property. If you set it to `'off'`, it does not log anything.
+ 
+Also, it is possible to connect any logging library to Hazelcast Node.js client through adapters.
```

Comment:
Notes:
- By "any", do we mean actually "any"? 
- Does Node.js client provide some built-in adapters? 
- I am guessing there is a default logger?

Reply:
I will change it as : `By default, there is a `Default Logger`. Also, it is possible to connect a custom logging library to Hazelcast Node.js client through adapters.`